crates/forge/tests/e2e/build_profile.rs

```diff
@@ -2,6 +2,7 @@ use super::common::runner::{setup_package, test_runner};
 use forge_runner::profiler_api::PROFILE_DIR;
 
 #[test]
+#[ignore]
```

Comment:
Make an issue to uncomment it when we release a new profiler

Reply:
https://github.com/software-mansion/cairo-profiler/issues/93